src/indexed_crate.rs

```diff
@@ -148,3 +166,142 @@ fn collect_public_items<'a>(
         Visibility::Crate | Visibility::Restricted { .. } => {}
     }
 }
+
+struct ManualTraitItem {
```

Comment:
```suggestion
#[derive(Debug)]
struct ManualTraitItem {
```
Essentially all types should be `Debug`, especially if it's as easy as adding a `derive`.